docs/html/development/getting-started.rst

```diff
@@ -38,13 +38,25 @@ To run the pip executable from your source tree during development, install pip
 locally using editable installation (inside a virtualenv).
 You can then invoke your local source tree pip normally.
 
-.. code-block:: console
+.. tabs::
+
+   .. group-tab:: Unix/macOS
+
+      .. code-block:: shell
```

Comment:
@shireenrao @pradyunsg I learned recently that there's a better option of terminal dialogues.
It should be `shell-session` and all the commands should have prompts. This highlighter knows to distinguish between commands and their outputs then, and highlights $-prompt separately

Reply:
@webknjaz - we have had that discussion above and the consensus was to not have prompts added. Initially I had `C:\>` for windows and `$` for linux/mac. I then took them all out.

Reply:
But it's present in many other files: https://github.com/pypa/pip/pull/8589/files#diff-25ae4181a2c6592acbec442969beb23cR131 still uses `console` and prompt, for example

Reply:
@webknjaz If you'd like to have an extended discussion on this, please file a new issue.

Reply:
No, just wanted to point out what I noticed and see if anybody cares :)